interpreters/src/show.rs

```diff
@@ -35,35 +41,38 @@ pub enum Error {
         backtrace: Backtrace,
     },
 
-    #[snafu(display("Create schema fail, err:{}", source))]
-    CreateSchema { source: crate::create::Error },
-
-    #[snafu(display("Show tables fail, err:{}", source))]
-    ShowTables { source: catalog::schema::Error },
-
-    #[snafu(display("Show database fail, err:{}", source))]
-    ShowDatabase { source: catalog::Error },
-
-    #[snafu(display("Arrow Record batch fail, err:{}", source))]
-    ArrowRecord {
+    #[snafu(display("Create a new arrow RecordBatch fail, err:{}", source))]
+    NewRecordBatch {
         source: arrow_deps::arrow::error::ArrowError,
     },
 
-    #[snafu(display("Arrow Record batch into fail, err:{}", source))]
-    ArrowInto {
+    #[snafu(display("Arrow RecordBatch to common_types::RecordBatch fail, err:{}", source))]
+    ToCommonRecordType {
         source: common_types::record_batch::Error,
     },
+
+    #[snafu(display("Fetch tables fail, err:{}", source))]
```

Comment:
I found other module's error message is like `Failed to ..., err:{}`, maybe we can follow this style.